tests/src/rules/no-commonjs.js

```diff
@@ -30,15 +29,15 @@ ruleTester.run('no-commonjs', require('rules/no-commonjs'), {
     },
 
     // allowed requires
-    { code: 'function a() { var x = require("y"); }' }, // nested requires allowed
```

Comment:
i prefer the object form in all cases; it reduces diff churn when later adding options.

Reply:
Sounds good, I reverted this change.